cspell.json

```
@@ -29,6 +29,8 @@
                 "Realloc",
                 "refcell",
                 "Rgba",
+                "smolstr",
+                "Smol",
                 "rtti",
                 "subspan",
```

I realized when commiting this is an unrelated change, but it doesn't hurt.
I whish i had gotten the alphabetic order correctly though
```suggestion
                "Rgba",
                "rtti",
                "Smol",
                "smolstr",
                "subspan",
```